docs/tx_lifecycle.md

```diff
@@ -0,0 +1,119 @@
+# Overview: The Lifecycle of an Arbitrum Transaction
+
+As an introduction to the various components that compromise the Arbitrum protocol, we'll go step-by-step over the phases an Arbitrum transaction goes through, starting with a client creating a signed transaction, to it ultimately being confirmed back on layer 1.
+
+We'll also intersperse it with "finality checks," explaining what guarantees the client has over their transaction's finality (i.e., assurances that their transaction's result is guaranteed and won't later be altered) over the course of a transaction's various stages.
+
+This overview will be focused on the Arbitrum Rollup protocol; see [Inside AnyTrust](inside_anytrust) for differences in the Arbitrum AnyTrust protocol. Also, for convenience/simplicity, we'll be describing the system in its fully permissionless final form; for the current state of the Arbitrum Mainnet chains, see "Mainnet Beta."
+
+For clarity on any terminology that may be unfamiliar see, "glossary."
+
+### 1. Sequencer receives transaction
+
+Typically, a transaction's lifecycle starts with the Sequencer, the entity designated with transaction ordering, receiving a transaction from a client. The Sequencer can receive a transaction one of two ways:
+
+##### 1a. Directly / Offchain
+
+For typical transacting within the L2 environment (i.e., using an L2 native dapp), a client will connect their wallet to an L2 node and directly deliver a signed transaction.
+
+##### 1b. ... or from L1 (via the Delayed Inbox).
+
+Alternatively, a client can send a message to the Sequencer by signing and publishing an L1 transaction in the Arbitrum chain's Delayed Inbox. This functionality is most commonly used for depositing ETH or tokens via a bridge.
+
+**See**:
+
+- [Retryables](arbos/ArbOS#retryables)
+- "The Sequencer"
+- "Token Bridge"
+
+### 2. Sequencer orders transaction (off-chain)
+
+Upon receiving a transaction, the Sequencer will:
+
+- Order it in its off-chain Inbox
+- Locally execute it using the Arbitrum Nitro VM (including collecting/allocating L1 and L2 fees, etc.)
+- "Instantly" give a transaction receipt to the client ("instant" in that it doesn't require any additional on-chain confirmations, and typically shouldn't take more than a second or two).
```

Comment:
I wouldn't raise that issue now, since we don't have a firm estimate for the BFT delay and we don't know if it will be large enough to be significant in practice.